src/blocks/TextBlock/nodes/ItemLink.test.ts

```diff
@@ -1,7 +1,7 @@
 import { describe, expect, test } from 'vitest';
```

Comment:
This test needs to be updated to match the new related component. As suggested in a comment on ItemLink.astro that component should maybe be split up first.